packages/shared/lib/core/profile/actions/active-profile/getBaseToken.ts

```diff
@@ -0,0 +1,9 @@
+import { BASE_TOKEN } from '@core/network/constants'
+import { activeProfile } from '@core/profile/stores'
+import { ITokenMetadata } from '@core/wallet'
+import { get } from 'svelte/store'
+
+export function getBaseToken(): ITokenMetadata {
```

Comment:
```suggestion
export function getBaseTokenMetadata(): ITokenMetadata {
```

Reply:
I don't agree with this, in a later pr I am replacing the where we derive base token with from the network itself so that its insync with whatever network you are connected to. And the nodeInfo already returns a baseToken object. I think we phase oout the ITokenMetadata in favour of an IToken in the future, that could be extended if needed.